### DISCUSSION TITLE
Issue with Open Type features in Figma

### DISCUSSION THREAD

**Discussion by FilipaGo** [rsms/inter#709] [Q&A]
Hi,
I'm having issues enabling some OTF properties in Figma. dlig only seems to work with Inter Display, not Inter, and I can't find a way to enable round quotes and commas. Some character variants like compact t and f are also not showing up in Figma.
Is anyone else having the same issues? Any solution?

**Comment by kenmcd** [comment]
Figma still has v3, which has fewer features than v4.
And it appears you are looking at the v4 features.
Old website for v3 features: https://d.rsms.me/inter-website/v3/#features
